packages/fxa-shared/subscriptions/configuration/helpers.ts

```diff
@@ -0,0 +1,164 @@
+import parser from 'accept-language-parser';
+
+import {
+  DEFAULT_PRODUCT_DETAILS,
+  metadataFromPlan,
+  productDetailsFromPlan,
+} from '../metadata';
+import { Plan } from '../types';
+import { ProductConfig } from './product';
+
+const DEFAULT_LOCALE = 'en';
+
+export const getPlanProductConfig = (plan: Plan) => {
+  const configuration = plan.configuration;
+
+  if (!configuration) {
+    throw new Error(`Plan configuration not found. (${plan.plan_id})`);
+  }
+
+  return configuration;
+};
+
+export const flattenProductConfig = (
```

Comment:
gonna nitpick a little on the name since it doesn't return a full config, just some localisable parts.

Reply:
What about `flattenProductConfigLocaleData`?